### ISSUE TITLE
Resolving will sometimes break, but not for every post.

### ISSUE THREAD

**Issue by coldReactive** [Lartsch/FediFollow-Chrome#77]
An example from kind.social
![image](https://github.com/Lartsch/FediAct/assets/1313673/a8fe25fa-5acf-4b4b-a879-9fc9a4c73287)

But on another tab viewing other posts from another instance, it's fine. Please put a hover/alt text on the element to indicate that it failed to resolved, like below:
![image](https://github.com/Lartsch/FediAct/assets/1313673/9ab0323e-e340-4b28-972f-4ed4159c2498)

I don't know to... ahem... _resolve_ this issue, no amount of refreshing or cache refreshing (CTRL+SHIFT+R) will fix it. Not ALL posts are like this however, see this boosted one from that user's account, which is also on kind.social:
![image](https://github.com/Lartsch/FediAct/assets/1313673/b6ac4cfb-d3ee-4ed5-b58a-cd776ae41902)

**Comment by coldReactive** [comment]
It seems like this is an ongoing issue with some features of Mastodon: https://github.com/mastodon/mastodon/issues/9770
